package.json

```diff
@@ -91,8 +92,8 @@
     "repl": "node --require ./scripts/repl.js",
     "fmt": "LOG_LEVEL= prettier-eslint --write --no-semi --ignore $PWD/'static/js/types/*.d.ts' $PWD/'static/js/**/*.ts' $PWD/'static/js/**/*.tsx'",
     "fmt:check": "LOG_LEVEL= prettier-eslint --list-different --no-semi --ignore $PWD/'static/js/types/*.d.ts' $PWD/'static/js/**/*.ts' $PWD/'static/js/**/*.tsx'",
-    "typescript": "rm -rf dist && rm -f .tsbuildinfo && tsc",
-    "typecheck": "tsc --noEmit",
+    "typescript": "rm -rf dist && rm -f .tsbuildinfo && tsc --strictNullChecks",
+    "typecheck": "tsc --noEmit --strictNullChecks",
```

Comment:
might be preferable to add this option to `tsconfig.json`, that way the config for typescript only lives in one place (also jest reads that config when it runs the tests)

Reply:
Looks like `strict` is already set to true so this option should be redundant